virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -10,6 +10,8 @@ A boot source contains a bootable operating system (OS) and all of the configura
 
 You use a boot source to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.
 
+Quick Start tours are available in the {product-title} web console to assist you in working with boot sources.
```

Comment:
I do not think this is specific enough. There is a Quick Start for [creating a custom boot source](https://gitlab.cee.redhat.com/cpaas-midstream/openshift-virtualization/hyperconverged-cluster-operator/-/blob/cnv-4.10-rhel-8/extras/quickStart/customize_a_boot_source_quickstart.yaml.in).

Reply:
Replaced with: _Quick Start tours are available in the {product-title} web console to assist you in creating a custom boot source, uploading a boot source, and other tasks._